### ISSUE TITLE
Using CLI with debug enabled uses wrong list type. Expected trending, seen anticipated.

### ISSUE THREAD

**Issue by MollyBe** [l3uddz/traktarr#100]
Enabling debug in the traktarr config causes unexpected results when importing movies with CLI. With debug enabled it will use the list type anticipated even though you specified trending. 

Run with debug set as false:
`traktarr movies -t trending -l 100 -s votes
`
and as expected it'll import movies from the trending list with output:
`Retrieved Trakt 'Trending' movies list, movies found: 1285
`

Run with debug set as true:
`traktarr movies -t trending -l 100 -s votes
`
and it'll unexpectedly import movies from the anticipated list type with output:

```
Request URL: https://api.trakt.tv/shows/anticipated
Retrieved Trakt 'Trending' movies list, movies found: 1017

```
Note: it says shows and not movies, and anticipated and not trending
I think it is still using the trending list but I cannot explain why the amount of movies found decreases when debug is enabled and more movies are imported. 

Reproducible using Master (latest).

Near the time of writing this trakt.tv had:
1002 in trending
1436 in anticipated

**Comment by desimaniac** [comment]
Try dev branch

**Comment by desimaniac** [comment]
It uses an arbitrary list to validate client ID info. 

https://github.com/l3uddz/traktarr/blob/3c89b265aafcfa62a2235894d9b49eaecb31c171/media/trakt.py#L236

**Comment by MollyBe** [comment]
Still the amount of results changes based on whether debug is on or off?

**Comment by desimaniac** [comment]
You did that on master 18 days ago. Master has since been updated, try it again. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by MollyBe** [comment]
Hi, I have tried it again with the same results as last time.

Command used:
`traktarr movies -t trending -l 1 -s votes`

With debug set to: _false_ 
The results were:
`1409 movies found, 6 to process, 0 added`

With debug set to: _true_ 
The results were:
``1312 movies found, 32 to process, at least 1 added``

Something unexpected is happening here. 

I have made a log file of this [here](https://pastebin.com/aP1rkzJ7).

Reproducible using Master (latest).
